README.md

```diff
@@ -54,16 +54,29 @@ module.exports = {
   daisyui: {
     themes: [
       "light",
-      // You can pick a rainbow color as your accent color, or not at all
-      catppuccin("latte", "sky"),
-      catppuccin("frappe", "yellow"),
-      catppuccin("macchiato", "peach"),
+      // You can simply select a catppuccin flavor with sane default colors
       catppuccin("mocha"),
+      // Or you can optionally customize your semantic colors with catppuccin defined color values, details on available values below
+      // Values not explicitly defined will use default values
+      catppuccin("latte", {primary: 'sky', secondary: 'rosewater'}),
     ],
   },
 };
 ```
 
+#### Available Catppuccin Flavors
+- Latte, Frappe, Macchiato, Mocha
+
+#### Customizable Semantic Colors
+- **Optional fields:** primary, secondary, accent, neutral, success, warning, error, info
```

Comment:
```suggestion
- **Optional fields:** primary, secondary, accent, neutral
```

To be honest I'm not sure if secondary or neutral should be able to be customised.